azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -44,13 +44,14 @@ def __init__(self, iothub_pipeline):
         self._edge_pipeline = None
 
     @classmethod
-    def create_from_connection_string(cls, connection_string, ca_cert=None):
+    def create_from_connection_string(cls, connection_string, ca_cert=None, **kwargs):
         """
         Instantiate the client from a IoTHub device or module connection string.
 
         :param str connection_string: The connection string for the IoTHub you wish to connect to.
         :param str ca_cert: The trusted certificate chain. Only necessary when using a
             connection string with a GatewayHostName parameter.
+        :**kwargs config: Set of keyword arguments for setting options.
```

Comment:
>:**kwargs config: Set of keyword arguments for setting options. [](start = 8, length = 63)

Right now the correct documentation (at least as best as the doc team has) is to list the optional arguments here, rather than using **kwargs. Which makes sense - **kwargs doesn't tell us anything useful.

Reply:
Since we have now agreed upon usage of **kwargs, I would want to see the individual options documented here

---
In reply to: [331168121](https://github.com/Azure/azure-iot-sdk-python/pull/331#discussion_r331168121) [](ancestors = 331168121)

Reply:
done.